resources/views/mini_views/navigational_view.blade.php

```diff
@@ -1,17 +1,23 @@
+<style>
+    .tab-content {
+    border:none;
+}
+</style>
 <div class="row">
     <div class="col col-md-3">
-        <div class="tabs tabs-vertical tabs-left">
+        <div class="tabs tabs-vertical tabs-left" style="border-top: 4px solid #0088CC;">
             <ul class="nav nav-tabs">
                 @foreach($navigations as $navigation)
                     <li class="nav-item {{isset($navigation['active'])? 'active': ''}}">
-                        <a class="nav-link" href="#{{$navigation['id']}}" data-toggle="tab">{{$navigation['name']}}</a>
+                       <a class="nav-link" href="#{{$navigation['id']}}" data-toggle="tab"> <i class="{{$navigation['icon']}}"> </i>&nbsp;{{$navigation['name']}}</a>
                     </li>
                 @endforeach
             </ul>
         </div>
     </div>
     <div class="col col-md-9">
-        <div class="tab-content" style="overflow-x: auto">
+        <div class="featured-box featured-box-primary text-left mt-5">
+        <div class="tab-content box-content" style="margin-top:-48px;overflow-x: auto">
```

Comment:
what is this for and how is it looking in mobile view?

Reply:
Fixed